.github/docker/Dockerfile.golang

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.19.2-bullseye
```

Comment:
Should this be 1.20.x ?

Reply:
Or is this to test the upgrade path?

Reply:
To test upgrades.

Reply:
:doh:

... Dockerfiles _will be_ updated by